package.json

```diff
@@ -50,6 +51,6 @@
     "typescript": "^5.4.5"
   },
   "dependencies": {
-    "@coinbase/cdp-agentkit-core": "^0.0.5"
```

Comment:
why does the monorepo depend on core? I think this should be removed.

Reply:
great callout - resolved!